content/_data/articles.js

```diff
@@ -0,0 +1,31 @@
+const groq = require('groq');
+const client = require('../../utils/sanityClient');
+const { hero, heroAlt } = require('../../utils/imageGroq');
+const toMarkdown = require('@sanity/block-content-to-markdown');
+const _ = require('lodash');
+
+module.exports = async function() {
+  return await client.fetch(groq`
+    *[_type == "article"
+      && !(_id in path("drafts.**"))
+    ]{
+      title,
+      subtitle,
+      summary,
+      body,
+      "slug": slug.current,
+      "tags": tags[]->title,
+      "date": _createdAt,
+      "category": category->title,
+      ${hero},
+    }
+  `).then(data => {
+    data.map(page => {
+      page.body = page.body ? toMarkdown(page.body) : '';
+      page.hero = heroAlt(page.hero);
+      return page;
+    });
```

Comment:
`Array.map()` doesn't alter the original array, and the returned value isn't being used -- is this block doing anything currently?